studio/LonaStudio/Workspace/FileNavigator.swift

```diff
@@ -185,6 +185,36 @@ class FileNavigator: NSBox {
                 menu.addItem(NSMenuItem.separator())
             }
 
+            if NSURL(fileURLWithPath: path).pathExtension == "component"{
```

Comment:
Nit: there should be a space after "component". I'm not sure why SwiftLint doesn't warn on that.

Reply:
Let's use `URL` instead of `NSURL`. I think they're aliases, but we use `URL` everywhere else.